source/Octopus.Client.Tests/Integration/OctopusClient/OctopusAsyncTestClient.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.IO;
+using System.Net;
+using System.Reflection;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Octopus.Client.Model;
+using Octopus.Client.Serialization;
+
+namespace Octopus.Client.Tests.Integration.OctopusClient
+{
+    public class OctopusAsyncTestClient : OctopusAsyncClient
```

Comment:
The repository code looks good. I'm not convinced that the `OctopusTestClient` provides any value over substituting `IOctopusClient`